src/components/pages/AboutPage/TeamMember.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FontAwesomeIcon } from '../../../utils/fontAwesome';
+import '../../../css/pages/AboutPage.css';
+
+class TeamMember extends React.Component {
+  static propTypes = {
+    name: PropTypes.string.isRequired,
+    title: PropTypes.string.isRequired,
+    favMovie: PropTypes.string.isRequired,
+    favShow: PropTypes.string.isRequired,
+    favGenre: PropTypes.string.isRequired,
+    imgUrl: PropTypes.string.isRequired
+  };
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      showInfo: false
+    };
+  }
+
+  showInfo = () => {
+    this.setState({
+      showInfo: true
+    });
+  };
+
+  hideInfo = () => {
+    this.setState({
+      showInfo: false
+    });
+  };
+
+  render() {
+    const { name, title, favMovie, favShow, favGenre, imgUrl } = this.props;
+    const { showInfo } = this.state;
+    return (
+      <div className="avatar-size-container">
+        <div className="avatar-container">
+          <div className="avatar-item">
+            <img src={imgUrl} alt={name} className="avatar-image" />
+          </div>
+          <h3 className="avatar-name">{name}</h3>
+          <h5 className="avatar-role">
+            {title} <br />
+            Software Engineer
+          </h5>
+          {showInfo ? (
+            <div className="show-info fade-in">
+              <p className="avatar-movie">
```

Comment:
If you do the array favorites prop as suggested above, you can just make this be a map. There's no real reason for the movie/show/genre sections to be different div tags, and you can just include any emojis in the text tags sorta like you already do for genre.

You can also do some checking and if `favorites.length < 1` just don't display any extra info or the expand button.

```suggestion
              this.props.favorites.map(fave => (<p className="avatar-text">{fave.type}: {fave.value}</p>)
```